src/nvim/fileio.c

```diff
@@ -7085,16 +7101,18 @@ int match_file_list(char_u *list, char_u *sfname, char_u *ffname)
   p = list;
   while (*p) {
     copy_option_part(&p, buf, 100, ",");
-    regpat = file_pat_to_reg_pat(buf, NULL, &allow_dirs, FALSE);
-    if (regpat == NULL)
+    regpat = file_pat_to_reg_pat(buf, NULL, &allow_dirs, false);
+    if (regpat == NULL) {
       break;
+    }
     match = match_file_pat(regpat, NULL, ffname, sfname,
         tail, (int)allow_dirs);
```

Comment:
Invalid alignment. Though linter will not complain (lines above and below did not change), it is better to fix this and `/*` comment above because this function is rather small and is already containing huge changes in this part.